apps/wallet-mobile/src/yoroi-wallets/cardano/constants/common.ts

```diff
@@ -57,3 +57,5 @@ export const MINIMUM_UTXO_VAL = '1000000'
 export const HISTORY_REFRESH_TIME = 25000
 
 export const COINS_PER_UTXO_WORD = '34482'
+
+export const COINS_PER_UTXO_BYTE = '4310'
```

Comment:
all these constants need to come from the chain, I'll ask BE team to update the tip status or create a specific endpoint for the protocol params.